invisible_cities/cities/detsim_waveforms_test.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+
+from invisible_cities.cities.detsim_waveforms import *
+
+
+
+def test_create_waveform():
```

Comment:
this is good, but you need to add maybe a more generic test with hypothesis and also extreme cases like those that you check within the function: sum is zero or nsamples is zero.

Also, the other two functions that are not tested at all!

Reply:
I added a test for Exception when `nsamples<1` and Warning for `nsamples>len(bins)` and for the conservation of total pes, that is sum of initial pes is equal to the sum of counts in waveform. Note this sum conservation is only true if times values lie within bin edges, if not, pes in the exterior are removed (recall conversation https://github.com/nextic/IC/pull/699#discussion_r390937048 which was solved in commit 
5962c8d)

Reply:
I agree that this test should stay, I meant that we need more :)

Reply:
Do you suggest any other necessary tests?